_posts/2017-01-04-OSS-FAQ.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Open Source FAQ for Engineers"
+date: 2017-01-04 12:17
+author: [orta, yayoi]
+categories: [oss, culture]
+series: Open Source by Default
+---
+
+We have a lot of Open Source code. For engineers without considerable experience in the open source realm, understanding some of the copyright issues around copyrights and code ownership can be tricky. I've been working with [dB](http://artsy.github.io/author/db/), our CTO and [Yayoi Shionoiri](http://www.theartgorgeous.com/yayoi-shionoiri/) (our Senior Counsel) on documenting how this works.
+
+I'd like to personally thank [Gil Yehuda](http://gilyehuda.strikingly.com) of Yahoo! for answering a lot of our questions, you really helped. For Artsy staff, you can get the living document at the [Engineering page on Artsy Atlas](https://sites.google.com/a/artsymail.com/intranet/team-pages/engineering).
+
+### What is Open Source?
+
+Open Source code is code that can be freely examined, used, adapted, and shared by all through a license that sets forth these principles.  The only potential limitation that an Open Source license is likely to impose is that future copies of the code (whether in adapted or un-adapted form) be themselves licensed in a manner consistent with the original license.  At Artsy, we are committed to making our engineering work Open Source by default.  A list of our Open Source projects can be [found here](http://artsy.github.io/open-source/).
+
+<!-- more -->
+
+### What is Open Source “by default”?
```

Comment:
Cool, good point, I've made the first mention of OSS by Default a link (its in the paragraph above )